packages/hub/node-tests/routes/data-integrity-checks-test.ts

```diff
@@ -82,28 +84,88 @@ describe('GET /api/data-integrity-checks/scheduled-payments', async function ()
       })
       .expect('Content-Type', 'application/vnd.api+json');
   });
+});
+
+describe('GET /api/data-integrity-checks/cron-tasks', async function () {
+  let db: DBClient;
+  type CronState = {
+    [key in TaskIdentifier]: { minutesAgo: number };
+  };
+  let { request, getContainer } = setupHub(this);
+
+  let setupKnownCrontabs = async (db: DBClient, task: CronState) => {
+    const tasks = Object.entries(task).map(async ([identifier, config]) => {
+      const query = `
+        INSERT INTO graphile_worker.known_crontabs (identifier, known_since, last_execution) 
+        VALUES ($1, current_timestamp, current_timestamp - interval '${config.minutesAgo} minutes');
+      `;
+      await db.query(query, [identifier]);
+    });
+
+    await Promise.all(tasks);
+  };
+
+  this.beforeEach(async function () {
+    let dbManager = await getContainer().lookup('database-manager');
+    db = await dbManager.getClient();
+    await getContainer().lookup('data-integrity-checks-cron-tasks');
+    await db.query('DELETE FROM graphile_worker.known_crontabs;');
```

Comment:
I think conceptually this would be better suited to be in `afterEach` - i.e. cleaning up after each test

Reply:
Or maybe you can even put the deletion in the beginning of the task setup function